static/bidder-params/resetdigital.json

```diff
@@ -1,13 +1,15 @@
 {
-    "$schema": "http://json-schema.org/draft-04/schema#",
-    "title": "Reset Digital Adapter Params",
-    "description": "A schema which validates params accepted by the Reset Digital adapter",
-    "type": "object",
-    "properties": {
-        "placement_id": {
-            "type": "string",
-            "minLength": 1,
-            "description": "Placement ID for the Reset Digital ad unit. This is the identifier for the ad unit on the Reset Digital platform, and its optional"
-        }
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "ResetDigital Adapter Params",
+  "description": "A schema which validates params accepted by the ResetDigital adapter",
+  "type": "object",
+  "properties": {
+    "placement_id": {
+      "type": "string",
+      "description": "Placement ID provided by ResetDigital",
+      "minLength": 1
     }
+  },
+  "required": ["placement_id"],
```

Comment:
Making a parameter required is a breaking change so we will need to hold this PR until the next major release which is 4.0. 4.0 will is tentatively scheduled to be released next month.